src/client/client.go

```diff
@@ -47,6 +47,8 @@ type APIClient struct {
 var (
 	//BlockSize is used internally by PFS blockserver
 	BlockSize = 8 * 1024 * 1024 // 8 Megabytes
+	// MaxBlockSize specifies the maximum block size for any data type
+	MaxBlockSize = 100 * 1024 * 1024 // 100 MB
 	// MaxMsgSize is used to define the GRPC frame size, which we need to be greater than a block
 	MaxMsgSize = 3 * BlockSize
```

Comment:
I see, would you mind terrible just decoupling them within this PR? It seems like it should be a pretty quick fix.

Also I dug into the grpc code for MaxMessageSize a bit and it seems like the MaxMsgSize only applies to incoming messages, not outgoing ones. Don't think that changes anything about this PR, just fyi.